coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -58,6 +60,8 @@ def __init__(self, language: str, docstyle: str, markers: (Iterable, str),
                                  'actually {}).'.format(length))
 
         self._metadata = metadata
+        self._top_padding = None if padding == () else int(padding[0])
+        self._bottom_padding = None if padding == () else int(padding[1])
 
```

Comment:
You can used NamedTuple to make it easier for them to be used.
The reason to keep them as a tuple is so that everywhere they are a pair, from style config to bear.  consistency.